### ISSUE TITLE
A question about adding h2gis and versions

### ISSUE THREAD

**Issue by sebasira** [orbisgis/h2gis#1195]
I have a doubt that I can not figure out.

Till now I've been adding it to my project (Gradle) like this:

```groovy
testCompile "org.orbisgis:h2gis-functions:1.3.2"
testCompile "org.orbisgis:h2gis-ext:1.3.2"
```

I'm currently updating all my dependencies and I saw that version **1.5.0** of h2gis is available, but when I replace the 1.3.2 for 1.5.0 it can not be found, also check maven central and latest versions where 1.3.2.

But then I saw in other places (blogs, forums, etc) that they include it like:

```groovy
testCompile "org.orbisgis:h2gis:1.5.0"
```

So I test that and all my test passed, but I don't know if I'm missinig something here.

I mean, is this the new way of adding it to a project? If not, what's the difference?

For what is worth, I'm working with `Spring Boot 2.4`, `h2 v1.4.197`, `geoDB v0.7` and `hibernateSpatial v5.4.25.Final`

Thank you very much!

**Comment by SPalominos** [comment]
It's normal, when we released the version 1.4.0, we changed the dependencies organization and naming in order to simplify the dependency management.
The two dependencies you where using (`h2gis-functions` and `h2gis-ext`) were both moved into the `h2gis` one. So now you just have to use 
```gradle
testCompile "org.orbisgis:h2gis:X.Y.Z"
```

**Comment by sebasira** [comment]
Thanks for the clarification!
